src/lib/components/magic/number-ticker/number-ticker.svelte

```
@@ -0,0 +1,117 @@
+
+      // Spring physics
+      const springForce = -stiffness * (position - target);
+      const dampingForce = -damping * velocity;
+      const acceleration = springForce + dampingForce;
+
+      velocity += acceleration * dt;
+      position += velocity * dt;
+
+      // Update display
+      if (spanRef) {
+        spanRef.textContent = `${prefix}${Intl.NumberFormat('en-US', {
```

ESLint

[error] Don't manipulate the DOM directly. The Svelte runtime can get confused if there is a difference between the actual DOM and the DOM expected by the Svelte runtime.

(svelte/no-dom-manipulating)